src/core/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/color-picker.css';
+import { CommonProps } from '../utils/props';
+import cx from 'classnames';
+
+export const DefaultColors = [
+  { color: '#FFFFFF', name: 'WHITE' },
```

Comment:
It was just like that where I copied it from. I can change it if it matters

Reply:
where did you copy it from?

Reply:
@gretanausedaite sent them to me, I'm not sure where she got them from.

Reply:
It seems that these random colors were pulled from the old bwc palette for prototyping purposes, and the names came from https://colornamer.robertcooper.me/ (which is why are they are uppercase).

I (and @FlyersPh9) think we will need to come up with a proper standard color list if we are going to export it to users.

For now I would suggest removing this list from the component and hardcoding it in the story. Unless @bentleyvk or @veekeys has a different opinion.

Reply:
I like these colors, Pallete looks nice and because its color picker its not that those need to be coming from standard colors. I know @FlyersPh9 worked on those a little, so not sure what is bad about it?

Reply:
It would be nice to release with predefined and approved colors so users wouldn't have to add some random colors. We currently have `SoftColors` in UX documentation site I see there is System colors that even has an example with color picker.

Reply:
> I know @FlyersPh9 worked on those a little, so not sure what is bad about it?

The work was for prototyping purposes and was never meant to be the final list we offer to users. The three biggest problems with the list are: names (random and uppercase), values (hex only), and count (why only 30 colors? why not 20 or 40?). Then there is also the problem of how the list is currently exported: it's hard to pick and choose from that list without knowing the index of each color.

> We currently have SoftColors

Those are not exported to the user.

> I see there is System colors in UX documentation site that even has an example with color picker.

There are not enough colors in that list. I think the current list is an extended version of that, with more shades from the bwc palette.

---

If we release with the current list, it would be hard to change it in the future. That's why we should make sure that it's proper if we want to export it.

Reply:
I can clean up the names to not be uppercase and the hex values to hsl. 
I can also change DefaultColors to be like SoftBackgrounds so that you can call the color by name (DefaultColors.White, DefaultColors.Grey) instead of by index only. 

I might need help coming up with some better color names, if that's something we want to change as well.

Reply:
If the names were neutral and a bit more useful, I could see this format working.
```ts
export const SystemColors = {
  white: {
    name: 'white',
    hex: '#FFFFFF',
    r: 255, g: 255, b: 255, rgb: 'rgb(255 255 255)',
    h: 0, s: 0.0, l: 100.0, hsl: 'hsl(0deg 0% 100%)',
  },
  // ...
} as const;
```

Would still need to curate the list with @FlyersPh9's help when he returns.